ui/raidboss/raidemulator.js

```diff
@@ -52,44 +51,29 @@ const Options = {
   let timelineController;
   let popupText;
   let persistor;
-  let logEventHandler;
   let encounterTab;
   let emulatedPartyInfo;
   let emulatedMap;
   let emulatedWebSocket;
-  let logConverter;
   let timelineUI;
+  let logConverterWorker;
 
   document.addEventListener('DOMContentLoaded', () => {
     emulator = new RaidEmulator(Options);
     progressBar = new ProgressBar(emulator);
     persistor = new Persistor();
-    logEventHandler = new LogEventHandler();
     encounterTab = new EncounterTab(persistor);
     emulatedPartyInfo = new EmulatedPartyInfo(emulator);
     emulatedMap = new EmulatedMap(emulator);
     emulatedWebSocket = new RaidEmulatorOverlayApiHook(emulator);
-    logConverter = new NetworkLogConverter();
+    logConverterWorker = new Worker('../../dist/raidemulator_worker.bundle.js');
```

Comment:
Here's the issue.  You named it `raidemulatorWorker` in `webpack.config.js` and it's trying to load `raidemulator_worker` instead here.

Reply:
I renamed it in the cjs file to camelCase because the linter didn't like the underscore, and forgot to update it in `raidemulator.js`. Apparently the old file was cached in my browser. Just force pushed the fix.

Reply:
Sorry, it's hard to tell the fix in the force push.  It looks like there's still a mismatch between `raidemulatorWorker` in the config and `raidemulator_worker` here.  I'll fix it and the conflict.